src/components/common/data-not-found/data-not-found.directive.coffee

```diff
@@ -0,0 +1,63 @@
+module = angular.module('impac.components.common.data-not-found',[])
+
+module.directive('commonDataNotFound', ($templateCache, $log, $http, ImpacAssets, ImpacTheming, ImpacMainSvc) ->
+  return {
+    restrict: 'A',
+    scope: {
+      isKpi: '=?'
+      engine: '='
+      widgetWidth: '=?'
+      onDisplayAlerts: '&?'
+    },
+    link: (scope, element) ->
+      scope.content     = ImpacTheming.get().dataNotFoundConfig
+      widgetImgsBaseDir = ImpacAssets.get('dataNotFound')
+      usingDefaults     = false
+      imagePath         = ''
+      image             = element.find('.data-not-found #not-found-bg')[0]
+
+      image.onerror = ->
+        missingImageLocationMsg = if usingDefaults then 'library defaults' else scope.engine
+        $log.warn("Missing data-not-found image for #{missingImageLocationMsg}")
```

Comment:
No, first I did extend the image capabilities for KPIs. But then I realised that an image background will not fit / look good with the KPIs. So I removed the ability. So indeed the image logic is bypassed for kpis.